src/components/topbar/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import Box from '@mui/material/Box';
-import AppBar from '@mui/material/AppBar';
-import Toolbar from '@mui/material/Toolbar';
+import { Box, AppBar, Toolbar, IconButton, Badge, Tooltip } from '@mui/material';
 import { LeftSide, RightSide, StyledContainer } from './styled';
+import { Person, Notifications, GridOnOutlined } from '@mui/icons-material';
```

Comment:
add a suffix 'Icons' to Icon components for better readability
e.g. PersonIcon/NotificationsIcon

Reply:
Updated. Will do the same for the sidebar as well 🤓 .